util/find-doc-nits

```diff
@@ -901,7 +901,7 @@ sub collectnames {
     }
     foreach my $name ( @{$podinfo{names}} ) {
         next if $name eq "";
-        err($id, "'$name' contains white space")
+        err($id, "'$name' contains whitespace")
```

Comment:
This change is wrong.

Reply:
???  what should it be?

Reply:
Uhm... sorry, my bad. Somehow I was confused and thought doc-nits was complaining about a "white space" misspelling, because I overlooked the trailing if.

Reply:
@mspncp, @levitte: Am curious about your thoughts on https://github.com/openssl/openssl/pull/12161#issuecomment-644390373